docs/glossary.toml

```diff
@@ -0,0 +1,84 @@
+["openpilot terms"]
+
+"big model".description = "A new paradigm in model development that takes a bigger input frame. Full frame is 1164x874, little model is a 512x256 crop, big model is a 1024x512 crop, 4x bigger than little. Make box bigger, drive better. Useful for signs and lights."
+
+"Driving Model".abbreviation = "model"
+"Driving Model".description = "The resulting neural network after Comma trains on driving data on their supercomputer.  This file lives on the device, and processes inputs to give outputs relevant to driving. Usually takes the form of an ONNX file, or a THNEED file after compilation on device. This file does not change or get trained on device, only processes inputs and outputs. See [the list of driving models](https://github.com/commaai/openpilot/wiki/Driving-Models) for names and details of models over time."
```

Comment:
Ideally we don't want to define terms like "Driving Model" multiple times. A map like this would be better

```
"openpilot terms" = [
  {term = "Driving Model", abbreviation = "model", description = "..."},
  {term = "End to end", abbreviation = "e2e", description = "..."}
]

"driver-assistance terms" = [
  {term = "Adaptive Cruise Control", abbreviation = "ACC", description = "..."}
]
```